src/test/integration/integrationtests.py

```diff
@@ -183,13 +183,9 @@ def test_resource(resource_url):
 
                 # If there are not only one open hours, current time should be in one of these open hours intervals
                 if len(open_hours[weekday]) > 1:
```

Comment:
I think this if statement is unnecessary now.

Reply:
ha you are right!